layouts/_default/baseof.html

```diff
@@ -23,6 +23,8 @@
     {{ if .Site.Params.cookie_consent_info_url }}
     {{ partial "cookie-consent.html" . }}
     {{ end }}
+
+		{{ partial "head-custom.html" . }}
```

Comment:
Nice change! I had the need too but did simply override cookie_consent.html. (It's changed anyway, so it was no problem for me).